docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+git stash
+git pull
```

Comment:
As commented below I have run it like this for the past month via cron and did not have a problem. But you are correct. Best practice would indeed be to fetch and checkout, but you won't be able to run it unattended.